meteor_packages/mats-common/public/python/ctc_stats.py

```diff
@@ -0,0 +1,156 @@
+import numpy as np
+
+error = ""
+
+
+# function for calculating critical skill index from MET contingency table counts
+def calculate_csi(fy_oy, fy_on, fn_oy):
+    global error
```

Comment:
I've had problems before with shadow variables in python, where there were three things with the same name running around and python gave me no indication whatsoever of the fact. This seemed less ambiguous, but if it's likely to cause other problems I will change it.

Reply:
Done.

Reply:
Ah, gotcha. Globals can lead to some really hard to debug 'spooky-action-at-a-distance' effects so I generally try to avoid them whenever possible. 

Variable shadowing can certainly be another issue - especially in Python since it can play fast & loose with the type system. One way to avoid that would be to use different variable names inside your functions. (e.g. - `error_msg` or `ctc_error`) However, unless you want to do that here, I don't think it's necessary.